pkg/network/netlink/socket.go

```diff
@@ -178,6 +178,17 @@ func (s *Socket) LeaveGroup(group uint32) error {
 	))
 }
 
+func (s *Socket) control(f func(int)) error {
+	rc, err := s.fd.SyscallConn()
```

Comment:
Isn't this already available as `s.conn`?

Reply:
Yes it is. Fixed.